pom_confluent.xml

```diff
@@ -324,29 +324,6 @@
                 </configuration>
             </plugin>
 
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-failsafe-plugin</artifactId>
-                <version>2.22.2</version>
-                <configuration>
-                    <includes>
-                        <include>**/*IT.java</include>
```

Comment:
might it be that case that your `-Dtest=!*Iceberg*IT'` param was ignored due to this setting?

Reply:
We have two plugins in our project: failsafe plugin (for integration tests) and surefire plugin (for unit tests). The previous version seems to be the flag for surefire but not for failsafe.